src/main/java/com/atlassian/db/replica/internal/circuitbreaker/BreakerConnection.java

```diff
@@ -22,121 +24,121 @@
 
 public class BreakerConnection implements Connection {
     private final Connection delegate;
-    private final BreakerHandler breakerHandler;
+    private final CircuitBreaker breaker;
```

Comment:
It looks weird when instance fields is called breaker, does it break something? Full name circutBreaker looks better ;)

Reply:
Fair point